service/internal/auth/casbin.go

```diff
@@ -44,6 +34,9 @@ var defaultPolicy = `
 # delete - delete the resource
 # unsafe - unsafe actions
 
+## Grouping Statements - Maps users/groups to roles
+g, opentdf-admin, role:admin
+g, opentdf-standard, role:standard
```

Comment:
i like the policy extending strategy instead of overriding, though what about the situation where the users wants to use these roles in their idp to mean different things? is it possible to change these lines when passing in your own policy? or is the default policy/groupings always there

Reply:
Yeah I added the ability to extend so they could do something like this.

```yaml
policy:
  extension: |
        g, my-group, role:admin
        g, casbin-user, role:standard
```

Or they could define their own roles

```yaml
policy:
  extension: |
        g, my-group, role:custom
        p, role:custom, policy.attributes.*, read, allow
```